README.md

```diff
@@ -42,10 +42,20 @@ Name     | Description | OS
 ---------|-------------|----
 bonding | Exposes the number of configured and active slaves of Linux bonding interfaces. | Linux
 devstat | Exposes device statistics | Dragonfly, FreeBSD
-gmond | Exposes statistics from Ganglia. | _any_
 interrupts | Exposes detailed interrupts statistics. | Linux, OpenBSD
 ipvs | Exposes IPVS status from `/proc/net/ip_vs` and stats from `/proc/net/ip_vs_stats`. | Linux
 ksmd | Exposes kernel and system statistics from `/sys/kernel/mm/ksm`. | Linux
+meminfo\_numa | Exposes memory statistics from `/proc/meminfo_numa`. | Linux
+tcpstat | Exposes TCP connection status information from `/proc/net/tcp` and `/proc/net/tcp6`. (Warning: the current version has potential performance issues in high load situations.) | Linux
+
+
+### Deprecated
+
+*These collectors will be (re)moved in the future.*
+
+Name     | Description | OS
+---------|-------------|----
+gmond | Exposes statistics from Ganglia. | _any_
 logind | Exposes session counts from [logind](http://www.freedesktop.org/wiki/Software/systemd/logind/). | Linux
 megacli | Exposes RAID statistics from MegaCLI. | Linux
 meminfo_numa | Exposes memory statistics from `/proc/meminfo_numa`. | Linux
```

Comment:
I was wondering the same when reading the diff.. Looks like this diff is just very confusing. If you look at the final file you'll see they aren't marked deprecated.